game.cpp

```diff
@@ -490,37 +481,33 @@ void Game::DummyGame()
 
 int Game::WritePlayers()
 {
-	Aoutfile f;
-	if (f.OpenByName("players.out") == -1) return(0);
-
-	f.PutStr(PLAYERS_FIRST_LINE);
-	f.PutStr(AString("Version: ") + CURRENT_ATL_VER);
-	f.PutStr(AString("TurnNumber: ") + TurnNumber());
+	ofstream f("players.out", ios::out|ios::ate);
+	if (!f.is_open()) return(0);
 
+	f << PLAYERS_FIRST_LINE << "\n";
+	f << "Version: " << CURRENT_ATL_VER << "\n";
+	f << "TurnNumber: " << TurnNumber() << "\n";
 	if (gameStatus == GAME_STATUS_UNINIT)
 		return(0);
-	else if (gameStatus == GAME_STATUS_NEW)
-		f.PutStr(AString("GameStatus: New"));
```

Comment:
Just to reply to your comment that you asked and then deleted.
Yes.

The previous thing was to write the status line (which put a newline at the end of it) and then below, just write a universal newline.  I just moved that extra newline up into each of the cases since they were fixed strings.

Reply:
Obviously verified also by the output not changing between the snapshots generated with the previous version of the code and the output from this code :)